libkpod/config.go

```diff
@@ -295,7 +292,6 @@ func DefaultConfig() *Config {
 			ContainerExitsDir: containerExitsDir,
 			HooksDirPath:      DefaultHooksDirPath,
 			LogSizeMax:        DefaultLogSizeMax,
```

Comment:
Do we need DefaultMounts in here?

Reply:
Nope, it will be set by the user crio.conf when they want to use it.

Reply:
Ok great, thx for the 411